pkg/virt-config/virt-config.go

```diff
@@ -131,6 +137,16 @@ func (c *ClusterConfig) GetFeaturesToDisable() map[string]*cmdv1.FeaturesToDisab
 		"Skylake-Server-noTSX-IBRS": {Features: []string{"mpx"}},
 		"Skylake-Server":            {Features: []string{"mpx"}},
 	}
+
+	cpuModelToFeaturesToDisableFromConfig := c.GetConfig().CpuFeaturesToDisable
+	for model, features := range cpuModelToFeaturesToDisableFromConfig {
+		var knownFeatures []string
+		if _, ok := cpuModelToFeaturesToDisable[model]; ok {
+			knownFeatures = cpuModelToFeaturesToDisable[model].Features
+		}
+		cpuModelToFeaturesToDisable[model] = &cmdv1.FeaturesToDisable{Features: mergeStringSlicesNoDuplications(knownFeatures, features)}
+	}
```

Comment:
Probably we should find a way to avoid this calculation for every VM scheduling

Reply:
Could you please elaborate? I'm not entirely sure I understand why we should avoid this.

Reply:
I mean, this merge between static features and kvConfiguration features lists is performed every time a VM is created, for every VM. I think that this should be done once: when the kubevirtCR is changed, and, during the VM creation it should only be read

Reply:
While I agree that performance is crucial and should be taken into consideration, there are a few points to note:

- The Kubevirt.spec.configuration.cpuFeaturesToDisable list will typically be empty by default. It's only intended for use in cases where there might be a vulnerability or a bug with one of the CPU features. Even then, the number of keys and values involved would be minimal, resulting in a very slight impact on performance.

- Implementing a cache for this map would introduce additional complexity and the potential for bugs.

- Even with a cached map, we'd still need to validate that the configuration hasn't changed, which could potentially be more resource-intensive than the initial calculation.

In summary, I don't believe that implementing such a cache would provide significant benefits. I'd love to hear your thoughts on this. 

Let me know what you think :=)